pkg/controller/endpointslicemirroring/endpointslicemirroring_controller.go

```diff
@@ -239,38 +241,39 @@ func (c *Controller) processNextWorkItem() bool {
 	}
 	defer c.queue.Done(cKey)
 
-	err := c.syncEndpoints(cKey.(string))
-	c.handleErr(err, cKey)
+	logger := klog.FromContext(context.TODO())
```

Comment:
Let's avoid *adding* TODO items as part of these changes - we want to *remove* them :grin: 

In this case the solution used elsewhere (check pending PRs) is to add a `logger` parameter pass that through from `Run`.

Reply:
this similar case is with another component and I added a comment [here](https://github.com/kubernetes/kubernetes/pull/115295#discussion_r1110193847)

Reply:
See https://github.com/kubernetes/kubernetes/pull/115295#discussion_r1132343721